File: day1/main.jl
f=open("input.txt","r")
lines=read(f,String)

function solve(lines::String)
    return eachmatch(r"((\d+\n)+)",lines) .|> (sum∘(x->parse.(Int32,x))∘split∘(x->x.match)) |> (x->sort(x,rev=true)) |> (x->(x[1],sum(x[1:3])))
end

display(solve())

File: day1/main.py
file=open("input.txt")

bags=[]
total=0
for line in file.readlines():
    if(line!="\n"):
        total+=int(line)
    else:
        bags.append(total)
        total=0
print(max(bags))
bags=sorted(bags)
print(bags[len(bags)-3:len(bags)])